app/controllers/api/my_predictions_controller.rb

```diff
@@ -3,25 +3,15 @@
 module Api
   class MyPredictionsController < AuthorisedController
     def index
-      render json: { user: @user.to_h, predictions: prediction_hash_array }
-    end
-
-    private
-
-    def prediction_hash_array
-      predictions.map do |prediction|
-        PredictionSerializer.new(prediction).serializable_hash
-      end
-    end
-
-    def predictions
-      PredictionsQuery.new(
+      ps = PredictionsQuery.new(
```

Comment:
```suggestion
      q = PredictionsQuery.new(
```

Reply:
Or `query`

Reply:
This line is initializing an instance of `PredictionsQuery`, invoking `call` on it to return a list of predictions that match the query parameters and then throws the instance away. So I think `q` is misleading.

Reply:
I agree - missed the `.call` at the end; too used to function calls appearing in prefix position.